samples/config.js

```diff
@@ -20,6 +20,7 @@ const defaults = {
 const spaDefaults = Object.assign({
   redirectPath: '/implicit/callback',
   flow: 'redirect',
+  scopes: 'openid email',
```

Comment:
I'm still not a fan of having scopes as a space-separated string here in the config. I would much prefer that scopes is an array, as it is for OktaAuth, and the conversion to string (comma or space separated) was done in the showForm() and conversion from string back to array is in loadConfig()